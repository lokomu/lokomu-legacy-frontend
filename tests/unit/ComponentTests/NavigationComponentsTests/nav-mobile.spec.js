import { shallowMount } from "@vue/test-utils";
import NavbarMobile from "@/components/NavigationComponents/NavMobile.vue";
import { describe, expect, it, beforeEach } from "vitest";

describe("NavBarMobile component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(NavbarMobile);
  });

  it("renders correctly", () => {
    expect(wrapper.element).toMatchSnapshot();
  });

  it("is instantiated", () => {
    expect(wrapper.exists()).toBeTruthy();
  });
});
