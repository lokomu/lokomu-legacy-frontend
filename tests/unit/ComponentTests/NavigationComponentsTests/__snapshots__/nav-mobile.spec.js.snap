// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NavBarMobile component > renders correctly 1`] = `
<div>
  <f-a-b-stub
    class="h-10 w-10 xs:h-12 xs:w-12 z-20 fixed left-1/2 -translate-y-1/2 -translate-x-1/2 bottom-0"
  />
  <!--v-if-->
  <section
    class="flex justify-between h-10 xs:h-12 z-10 fixed inset-x-0 bottom-0 bg-transparent overflow-hidden"
  >
    <div
      class="w-full flex justify-between h-10 xs:h-12 z-20 rounded-tr-3xl bg-gradient-to-r from-pink-500 to-green-500 dark:from-pink-1000 dark:to-green-1000"
    >
      <div
        class="w-full cursor-pointer flex"
      >
        <logo-icon-stub
          class="m-auto w-9 fill-white"
        />
      </div>
      <div
        class="w-full cursor-pointer flex"
      >
        <borrow-icon-stub
          class="h-8 m-auto fill-white"
        />
      </div>
    </div>
    <div
      class="justify-between flex"
    >
      <div
        class="h-24 w-20 xs:w-24 bg-green-500 dark:bg-green-1000 -rotate-45 translate-x-[1.5px] xs:-translate-x-[4.66px] rounded-tr-md"
      />
      <div
        class="w-full"
      />
      <div
        class="h-24 w-full bg-green-500 dark:bg-green-1000 rotate-45 translate-x-[-1.5px] xs:translate-x-[4.66px] rounded-tl-md"
      />
    </div>
    <div
      class="w-full flex justify-between h-10 xs:h-12 z-20 rounded-tl-3xl bg-gradient-to-r from-green-500 to-floyd-500 dark:from-green-1000 dark:to-floyd-1000"
    >
      <div
        class="flex w-full relative cursor-pointer"
      >
        <chat-icon-stub
          class="m-auto fill-white h-8"
        />
      </div>
      <div
        class="w-full flex cursor-pointer"
      >
        <menu-icon-stub
          class="m-auto fill-white h-8"
        />
      </div>
    </div>
  </section>
  <!--v-if-->
</div>
`;
